src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/media/fixtures.js

```diff
@@ -0,0 +1,101 @@
+export const CPSMediaBlock = {
+  id: 'p07jl3lv',
+  subType: 'clip',
+  format: 'video',
+  title: 'Why dis pikin get fish skin',
+  synopses: {
+    short: 'Short Synopsis',
+    long: 'Long Synopsis',
+    medium: 'Medium Synopsis',
+  },
+  imageUrl: 'ichef.bbci.co.uk/images/ic/$recipe/p07jl8g4.jpg',
+  embedding: true,
+  advertising: true,
+  caption: 'Warning: Dis video fit make you feel one kain!',
+  versions: [
+    {
+      versionId: 'p07jl3lx',
+      types: ['Original'],
+      duration: 101,
+      durationISO8601: 'PT1M41S',
+      warnings: {
+        long: 'This is a long warning',
+        short: 'This is a short warning',
+      },
+      availableTerritories: {
+        uk: true,
+        nonUk: true,
+      },
+      availableFrom: 1564740166,
+    },
+  ],
+  imageCopyright: 'BBC',
+  type: 'media',
+};
+
+export const optimoVideoBlock = {
+  type: 'video',
+  model: {
+    locator: 'urn:bbc:pips:pid:p07jl3lv',
+    blocks: [
+      {
+        type: 'aresMedia',
+        model: {
+          blocks: [
+            {
+              type: 'aresMediaMetadata',
+              blockId: 'urn:bbc:ares::clip:p07jl3lv',
+              model: {
+                advertising: true,
+                embedding: true,
+                format: 'video',
```

Comment:
This should be `audio_video`, not sure if it matters, but it might for some existing container logic, worth checking

Reply:
👍 changed to audio_video